TLM/TLM/UI/SubTools/LaneArrowTool.cs

```diff
@@ -321,16 +321,17 @@ public LaneArrowTool(TrafficManagerTool mainTool)
                 if (buttonClicked) {
                     switch (res) {
                         case SetLaneArrowError.HighwayArrows: {
-                            MainTool.ShowError(
-                                Translation.LaneRouting.Get("Dialog.Text:Disabled due to highway rules"));
-                            break;
-                        }
-
+                                MainTool.Guide.Activate("Disabled due to highway rules");
+                                break;
+                            }
                         case SetLaneArrowError.LaneConnection: {
-                            MainTool.ShowError(
-                                Translation.LaneRouting.Get("Dialog.Text:Disabled due to manual connection"));
+                                MainTool.Guide.Activate("Disabled due to lane connections");
+                                break;
+                            }
+                        case SetLaneArrowError.Success:
+                            MainTool.Guide.Deactivate("Disabled due to highway rules");
+                            MainTool.Guide.Deactivate("Disabled due to lane connections");
```

Comment:
Seems to be same as lines 50-62 - maybe put in to separate method and call it from both places?